ovn-tester/ovn_load_balancer.py

```diff
@@ -11,8 +11,21 @@ def __str__(self):
         return f"Invalid Protocol: {self.args}"
 
 
+async def create_load_balancer(lb_name, nbctl, vips=None,
```

Comment:
Thanks for the suggestion. I thought about doing it this way. In the end, I went with the method in this PR because it seemed less likely to lead to errors.

For example, if we go with the suggested method, then if I wish to create a load balancer, I'd do something like this:
```
from ovn_load_balancer import OvnLoadBalancer
...
load_balancer = await OvnLoadBalancer.create(...)
```
However, because the entire `OvnLoadBalancer` class is imported, it would still be possible to do something like
```
load_balancer = OvnLoadBalancer(...)
```
This would result in a partially initialized load balancer being created and likely causing errors later. Best of luck to anyone trying to debug this :) With the method I used, it allows for this pattern:
```
from ovn_load_balancer import create_load_balancer
...
load_balancer = await create_load_balancer(...)
```
With this import, it is not possible to construct a load balancer directly.

There is nothing stopping people from importing the entire `ovn_load_balancer` file or the `OvnLoadBalancer` class with my method. I'm hopeful that the precedent set in this PR would be followed, though.

I'm willing to change over to using a `staticmethod` factory method if there are reasons for doing so that I'm not seeing. I figured I'd plead my case here first.

I really wish Python supported private methods.

Reply:
It's ok, thanks for the rationale, let's leave it as is.